AppInspector/Commands/TagDiffCommand.cs

```diff
@@ -11,12 +11,14 @@ namespace Microsoft.ApplicationInspector.Commands
 {
     public class TagDiffOptions : CommandOptions
```

Comment:
This doesn't seem to be exactly TagDiffs, since TagDiffs would contain added and removed tags. More like newTags/TagsAdded?

Reply:
Thanks for pointing this out.  THis is supposed to create the diffs.  I've fixed it.